### PR TITLE
update repo validation texts

### PR DESCRIPTION
**Pull request by abhinandan13jan**
## Fixes 

https://issues.redhat.com/browse/RHTAPBUGS-67

## Description

- Removes help popover
- Updates validation text for incorrect URL
- Updates validation text for private repository

## Type of change

- [ ] Feature
- [x] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Screen shots / Gifs for design review 
Screenshot 2023-05-24 at 5 01 52 PM
Screenshot 2023-05-24 at 5 02 07 PM

## Browser conformance: 

- [x] Chrome
- [x] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: src/components/ImportForm/SourceSection/SourceSection.tsx [hunk 1]
```diff
@@ -94,7 +93,7 @@ const SourceSection: React.FC<SourceSectionProps> = () => {
           const { organization } = gitUrlParse(sourceUrl);
           if (!organization) {
             setValidated(ValidatedOptions.error);
-            setHelpTextInvalid('Not a valid source repository');
+            setHelpTextInvalid("That repository URL isn't quite right. Try again.");
```
Comment (rohitkrai03) [inline src/components/ImportForm/SourceSection/SourceSection.tsx:96]: You also need to update the text in validation utils.

**Review by rohitkrai03 — CHANGES_REQUESTED** [review]

File: src/components/ImportForm/SourceSection/SourceSection.tsx (hunk 1 above)
Comment (abhinandan13jan) [reply src/components/ImportForm/SourceSection/SourceSection.tsx:98]: added

**Review by rohitkrai03 — APPROVED** [review]
/lgtm

**Pull request merged**